### PR TITLE
Add triage-wg to release team

### PR DESCRIPTION
**Pull request by Dylan-DPC**

### CODE REVIEW

File: teams/wg-triage.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name = "wg-triage"
+subteam-of = "release"
+wg = true
+
+[people]
+leads = []
```
Comment (Centril) [inline teams/wg-triage.toml]: Shouldn't the lead be you?

File: people/soniasingla.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name = "Sonia"
+github = "soniasingla"
+email = "soniasingla.1812@gmail.com"
+
+[permissions]
+bors.compiler_builtins.review = false
```
Comment (Mark-Simulacrum) [inline people/soniasingla.toml]: Presumably this isn't necessary, as nothing is granting all of these members this permission?

Comment (pietroalbini) [reply people/soniasingla.toml]: Yeah, this is not needed.

File: teams/wg-triage.toml (hunk 1 above)
Comment (Dylan-DPC) [reply teams/wg-triage.toml]: oops yeah 😄

File: teams/wg-triage.toml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+members = [
+    "Dylan-DPC",
+    "fairingrey",
+    "gagan0723",
+    "JohnTitor",
+    "Kixiron",
+    "joelpalmer",
+    "hdhoang",
+    "johncsimon",
+    "edmilsonefes",
```
Comment (JohnTitor) [inline teams/wg-triage.toml]:
```suggestion
    "edmilsonefs",
```

**Pull request merged**